Index: app/src/main/java/com/example/ecard/ui/edit/EditScreen.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/ecard/ui/edit/EditScreen.kt b/app/src/main/java/com/example/ecard/ui/edit/EditScreen.kt
--- a/app/src/main/java/com/example/ecard/ui/edit/EditScreen.kt	
+++ b/app/src/main/java/com/example/ecard/ui/edit/EditScreen.kt	
@@ -155,6 +155,7 @@
                                 .width(100.dp)
                                 .clickable { homeViewModel.onPickSocialItem(5) }
                         )
+
                     }
 
                     Row(
@@ -200,6 +201,7 @@
                     social = homeViewModel.currentPickSocial.value!!,
                     onCancelClick = { homeViewModel.onCancelOrDismissClickPopup() },
                     onSaveClick = {
+
                     }
                 )
             }
